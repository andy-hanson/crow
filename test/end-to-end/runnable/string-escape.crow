main void()
	info log "\xe1 is Ã¡"
	info log "\u30c3 is ãƒƒ"
	info log "\U0001f63c is ðŸ˜¼"
