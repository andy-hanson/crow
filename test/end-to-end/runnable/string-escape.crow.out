info: รก is รก
info: ใ is ใ
info: ๐ผ is ๐ผ
